application.py

```diff
@@ -6,7 +6,8 @@
 
 application = create_app(os.getenv('DM_ENVIRONMENT') or 'development')
 manager = Manager(application)
-manager.add_command("runserver", Server(port=5002))
+manager.add_command("runserver", Server(
+    os.getenv('DM_HOST') or '127.0.0.1', port=5002))
```

Comment:
This works for me.  Are we 100% sure that '127.0.0.1' is the right value for when it's deployed to AWS?